src/Header.js

```diff
@@ -59,58 +63,50 @@ export default function Header(props) {
                   color="inherit"
                   component={Logo}
                   viewBox={"0 0 1000 169.32"}
-                  sx={{ height: "28px", width: "166px" }}
+                  sx={{ height: "32px", width: "auto" }}
                 />
               </IconButton>
             </Grid>
 
-            <Grid item justifyContent="flex-end">
-              {/*<Button
-                href="#products"
-                variant={`text-${colorMode}`}
-                sx={{ minWidth: 0 }}
-              >
-                Products
-              </Button>
+            <Grid
+              item
+              justifyContent="flex-end"
+              marginTop={{ xs: "20px", sm: "0" }}
+            >
               <Button
-                href="#partners"
-                variant={`text-${colorMode}`}
-                sx={{ minWidth: 0 }}
-              >
-                Partners
-              </Button>
-              <Button
-                target="_blank"
-                href="https://github.com/keeperdao/docs/tree/master/audits"
+                href="./about"
                 rel="noopener noreferrer"
                 variant={`text-${colorMode}`}
-                sx={{ minWidth: 0 }}
+                sx={{
+                  minWidth: 0,
+                  fontWeight: 600,
+                }}
               >
-                Audits
+                About
               </Button>
               <Button
-                target="_blank"
                 href="https://blog.keeperdao.com/"
                 rel="noopener noreferrer"
                 variant={`text-${colorMode}`}
-                sx={{ minWidth: 0 }}
+                sx={{
+                  minWidth: 0,
+                  fontWeight: 600,
+                }}
               >
                 Blog
               </Button>
-              */}
               <Button
+                href="https://app.keeperdao.com/"
+                rel="noopener noreferrer"
                 variant={`outlined-${colorMode}`}
                 sx={{
-                  marginLeft: "8px",
-                  height: "32px",
-                  width: "64px",
+                  marginLeft: "16px",
+                  height: "38px",
+                  width: "auto",
                   borderRadius: "8px",
                 }}
-                target="_blank"
-                href="https://app.keeperdao.com/"
-                rel="noopener noreferrer"
               >
-                App
+                Enter App
```

Comment:
```suggestion
                App
```
I don't think we need the word "enter" here, "blog - abut - app" is fine

Reply:
on second thoughts, this is fine